data/mods/classic_zombies/items/tools.json

```
@@ -0,0 +1,52 @@
+[
+  {
+    "id": "portable_music_player"
+    "type": "ITEM",
+    "subtypes": [ "TOOL" ],
+    "name": { "str": "portable cassette player (off)", "str_pl": "portable cassette players (off)" },
+    "description": "This battery-powered device can be used to play cassettes.  Fortunately, you have a couple on you, and listening to them will raise your morale slightly.  Use it to turn it on.",
```

```suggestion
    "description": "This battery-powered device can be used to play cassettes (and like any good survivor, you've always got your favorites at hand).  Listening to them will raise your morale slightly.  Use it to turn it on.",
```
Modifying to lampshade a little further, because I think it's funny.